jax/_src/lax/lax.py

```diff
@@ -69,6 +69,23 @@
 DType = Any
 Shape = core.Shape
 
+def _array_copy(arr):
```

Comment:
Could you put a `jit` on this? That way XLA will optimize away the addition.

Reply:
Good call - done!